package.json

```diff
@@ -31,7 +31,10 @@
     "Linked Data"
   ],
   "devDependencies": {
-    "eslint": "^7.3.1",
-    "eslint-config-digitalbazaar": "^2.3.0"
+    "eslint": "^7.22.0",
+    "eslint-config-digitalbazaar": "^2.6.1"
+  },
+  "engines": {
+    "node": ">=14"
   }
```

Comment:
Looks like `engines` is being added here which is not needed in a `web` package which is browser only code.

Reply:
Removed here 10ac153.